src/System.CommandLine/CommandExtensions.cs

```diff
@@ -42,11 +42,12 @@ public static class CommandExtensions
 
         private static InvocationPipeline GetInvocationPipeline(Command command, string[] args)
         {
-            var parser = new CommandLineBuilder(command)
-                         .UseDefaults()
-                         .Build();
+            var parser = command.ImplicitParser ??=
```

Comment:
You shouldn't need to explicitly set `ImplicitParser` here. Calling `.Build` will set it on the command passed into the builder.
```suggestion
            var parser = command.ImplicitParser ??
```